ui/src/common/components/APIButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ApiOutlined } from '@ant-design/icons';
+
+import IconText from './IconText';
+import LinkWithTargetBlank from './LinkWithTargetBlank';
+import UserAction from './UserAction';
+
+export const APIButton = ({ url }: { url: string }) => {
+  const transformUrl = (url: string): string => {
+    const urlParts = url.split('/');
+    urlParts.splice(3, 0, 'api');
+    return urlParts.join('/');
```

Comment:
it’s more safe to use something like the following :)

```suggestion
  const transformUrl = (url: string): string => {
     const parsedUrl = new URL(url);
  const pathname = parsedUrl.pathname;
  const newPathname = `/api${pathname}`;
  parsedUrl.pathname = newPathname;
  return parsedUrl.href;
 }
```